frequency_calculator.py

```diff
@@ -0,0 +1,36 @@
+from pathlib import Path
```

Comment:
Disagree with the structure.
You've put all the files simply in the main directory and all the logic is placed there.
Please read about the MVC architecture. I want the code to be structured with
1. Clear entrypoint that does nothing but creating entrypoint to the cli application.
2. CLI command that is responsible for accepting parameters, maybe logs some generic messages and calls Controller logic

Reply:
I made necessary update and I will be glad if you take another look. Thank you!